graphite_api/utils.py

```diff
@@ -90,6 +90,15 @@ def to_seconds(delta):
     return abs(delta.seconds + delta.days * 86400)
 
 
+def deltaseconds(timedelta):
+    """
+    Convert a timedelta object into seconds
+    (same as timedelta.total_seconds() in Python 2.7+)
```

Comment:
Actually, I don't know how I missed that `to_seconds` does the exact same thing as this!